x-pack/plugins/alerting/server/rules_client/lib/validate_actions.ts

```diff
@@ -22,7 +22,11 @@ export async function validateActions(
   const hasNotifyWhen = typeof notifyWhen !== 'undefined';
   const hasThrottle = typeof throttle !== 'undefined';
   let usesRuleLevelFreqParams;
-  if (hasNotifyWhen && hasThrottle) usesRuleLevelFreqParams = true;
+  // I removed the below ` && hasThrottle` check temporarily.
+  // Currently the UI sends "throttle" as undefined but schema converts it to null, so they never become both undefined
+  // I changed the schema too, but as the UI (and tests) sends "notifyWhen" as string and "throttle" as undefined, they never become both defined.
+  // We should add it back when the UI is changed (https://github.com/elastic/kibana/issues/143369)
+  if (hasNotifyWhen) usesRuleLevelFreqParams = true;
```

Comment:
No there isn't any breaking change. But it's a bit confusing, took me some time to figure out as well...

I want to test the new frequency fields by creating a new rule with `notifyWhen` and `throttle` in action level. 
But as per the above code both rule level `notifyWhen` and `throttle` should be undefined.
As the current schema converts `undefined` throttle field value to `null` (schema.nullable -> schema.maybe order) it was impossible, so i changed it. 
Then functional tests started to fail because the current UI sends `notifyWhen` as string and `throttle` as undefined! So the following if case never returns true `if (hasNotifyWhen && hasThrottle) usesRuleLevelFreqParams = true;`

So i removed the `hasThrottle` part...

Actually, as per our design, when there are `notifyWhen` and `throttle` on action level, rule level ones should be null, not undefined... but we can tackle that when we implement the UI part of the meta...